vendor/bat-native-ledger/src/bat/ledger/internal/bat_helper.cc

```diff
@@ -1093,13 +1093,13 @@ bool RECONCILE_DIRECTION::loadFromJson(const std::string & json) {
   // has parser errors or wrong types
   bool error = d.HasParseError();
   if (!error) {
-    error = !(d.HasMember("amount") && d["amount"].IsInt() &&
+    error = !(d.HasMember("amount") && d["amount"].IsDouble() &&
```

Comment:
we need to be backward compatible as old file could have int in there. So we need to check if int or double

Reply:
Fixed!